cpu/esp_common/esp-wifi/esp_wifi_netdev.c

```diff
@@ -903,6 +904,62 @@ static wifi_config_t wifi_config_ap = {
 };
 #endif /* (defined(MCU_ESP8266) && !defined(MODULE_ESP_NOW)) || defined(MODULE_ESP_WIFI_AP) */
 
+
+#if (defined(MCU_ESP32) && defined(MODULE_ESP_WIFI_CSI))
+void __attribute__((weak)) esp_wifi_csi_rx_cb(void *ctx, wifi_csi_info_t *info)
+{
+    if (!info || !info->buf) {
+        ESP_WIFI_LOG_ERROR("ERROR: esp_wifi_csi_rx_cb missing info or buf");
+        return;
+    }
+    printf("{");
+    printf("\"mac\": \"" MAC_STR "\",", MAC_STR_ARG(info->mac));
```

Comment:
Is it really necessary to put the MAC address in quotes?
```suggestion
    printf("\"mac\": " MAC_STR ",", MAC_STR_ARG(info->mac));
```